orcid-web/src/main/webapp/static/javascript/script.js

```diff
@@ -129,24 +129,31 @@ var OrcidCookie = new function() {
     };
 };
 
-var _gaq = _gaq || [];
-
 var OrcidGA = function() {
     // test and make sure _gaq is working. disconnect.me chrome plugin has
     // caused silent _gaq failures. This check allows us to detect that
     // situation
     var gaEnabled = false;
```

Comment:
I'm not 100% sure how I was making sure the GA script was loaded before this code was run....I think we should make a method retest the and reset the gaEnabled variable on load of the GA script.